cmd/ical-relay/templates/calendar.html

```diff
@@ -111,12 +111,12 @@
                 calendarEnd = date.add(4, "week").day(6);
             }else{
                 document.getElementById("current-date").innerHTML = "KW"+currentDate.format(" w MMMM");
-
                 calendarStart = date.startOf("week").subtract(1, "day");
+                if(date.day() === 0){
+                    calendarStart = calendarStart.add(1,"week")
+                }
```

Comment:
Because this modifies the timeframe outlined by calendarStart and calendarEnd to not include currentDate (selectedDate), this causes the wrong calendarWeek to be displayed in the title and also the location hash to be wrong (although it is debatable if this is bad).
I think the best solution is to just move this check into the beginning of the function setSelectedDate, which also eliminates the need for the change on line 144.